tests/specs/v2/rumXhrStatusTracking.js

```diff
@@ -75,6 +75,28 @@ describe("RUM status code tracking", function() {
     checkStatusCodes();
   });
 
+  describe('with the global window.fetch objects saved as a reference', () => {
+    describe('and using the minified.fetchhandler.js code snippet', () => {
+      it('attaches status codes to requests', () => {
+        browser.url('http://localhost:4567/fixtures/v2/rumReferencedFetchWithFetchSnippet.html');
+
+        browser.pause(1000);
+    
+        var supportsFetch = browser.execute(function() {
+          return window.supportsFetch;
+        }).value;
+    
+        if (!supportsFetch) {
```

Comment:
Is this statement necessary? Would `supportsFetch` ever be false? If it is, does that affect the outcome of the test?

Reply:
The tests support being run against legacy browsers before fetch was supported at all. `supportsFetch` is generated by the fixtures and could be false for legacy IE versions we test against.